packages/apollo-engine-reporting/src/agent.ts

```diff
@@ -60,6 +60,8 @@ export interface EngineReportingOptions {
   endpointUrl?: string;
   // If set, prints all reports as JSON when they are sent.
   debugPrintReports?: boolean;
+  // HTTP(s) agent to be used on the fetch call to apollo-engine metrics endpoint
+  requestAgent?: RequestAgent | boolean;
```

Comment:
What does the `boolean` here do? What does it mean to pass in `true` or `false`?